src/utils/parseCalloutTags.ts

```diff
@@ -5,19 +5,8 @@ import CalloutBlock from '../components/CalloutBlock';
  * Turn [hint] and other callouts into a proper React component.
  * @param data
  */
-const parseCalloutTags = (data: any): ReactElement|false => {
-    let matches;
-
-    matches = data.match(/\[(hint|warning|info|alert|notice|note)\](.*?)\[\/(?:hint|warning|info|alert|notice|note)\]/s);
-
-    if (matches) {
-        const type = matches[1];
-        const content = matches[2];
-
-        return createElement(CalloutBlock, { type, content });
-    }
-
-    return false;
+const parseCalloutTags = (type: string, content: any): ReactElement|false => {
+    return createElement(CalloutBlock, { type, content });
```

Comment:
We don't need to double up the regex anymore, so this ends up being a lot simpler.